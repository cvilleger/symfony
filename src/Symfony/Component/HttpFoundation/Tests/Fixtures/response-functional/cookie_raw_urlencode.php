<?php

use Symfony\Component\HttpFoundation\Cookie;

/** @var \Symfony\Component\HttpFoundation\Response $r */
$r = require __DIR__ . '/common.inc';

$url = 'https://blackfire.io/?*():@&+$/%#[]';

$r->headers->setCookie(new Cookie($url, $url, 0, '/', null, false, false, true));
$r->sendHeaders();

setrawcookie($url, $url, 0, '/', null, false, false);
