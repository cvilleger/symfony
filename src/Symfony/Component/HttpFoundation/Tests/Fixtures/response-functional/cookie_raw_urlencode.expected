
Array
(
    [0] => Content-Type: text/plain; charset=utf-8
    [1] => Cache-Control: no-cache, private
    [2] => Date: Sat, 12 Nov 1955 20:04:00 GMT
    [3] => Set-Cookie: https://blackfire.io/?*():@&+$/%#[]=https://blackfire.io/?*():@&+$/%#[]; path=/
    [4] => Set-Cookie: https://blackfire.io/?*():@&+$/%#[]=https://blackfire.io/?*():@&+$/%#[]; path=/
)
shutdown
